lib/src/util/component_usage_migrator.dart

```diff
@@ -433,6 +433,23 @@ abstract class ComponentUsageMigrator with ClassSuggestor {
     yieldPatch('', start, end);
   }
 
+  /// Yields a patch that adds a given [child] to a usage.
+  void yieldAddChildPatch(FluentComponentUsage usage, String child) {
```

Comment:
It's not immediately obvious from the name of this method or the doc comment where the child is inserted